pkg/didcomm/protocol/presentproof/service.go

```diff
@@ -524,6 +522,10 @@ func (s *Service) ActionStop(piID string, cErr error) error {
 		return fmt.Errorf("delete transitional payload: %w", err)
 	}
 
+	if cErr == nil {
+		cErr = errProtocolStopped
+	}
+
 	md.err = customError{error: cErr}
```

Comment:
same here, if cErr!= nil, shouldn't we wrap `errProtocolStopped` to it? .. I see this repeated elsewhere in this pr.